datadog_checks_base/datadog_checks/base/utils/subprocess_output.py

```diff
@@ -64,9 +64,7 @@ def get_subprocess_output(command, log, raise_on_empty_output=True, log_debug=Tr
         )
     )
 
-    if out:
-        out = ensure_unicode(out)
-    if err:
-        err = ensure_unicode(err)
+    out = ensure_unicode(out)
+    err = ensure_unicode(err)
```

Comment:
Do we still need to check for `None` ?

Reply:
Yeah, lets. The subprocess command seems to actually come from some binding in the Agent, so better to be safer than sorry here.